main.py

```diff
@@ -7,9 +7,9 @@
 
 from Major_Functions import *
 
-print("-----------------------------------------------")
-print("-----------------Library App-------------------")
 
+print("-----------------Library App-------------------")
+print("this is a copy")
```

Comment:
your forked repo is the copy, not the original repo :). If I accept your pull request, then the main code will contain 'this is a copy'.